Documentation/AppendixA/Index.rst

```diff
@@ -182,16 +182,54 @@ writing: :php:`TypoScriptFrontendController->id`.
    Variable
          fe\_user
 
-   PHP-Type
-         object
-
-   Description
-         The current front-end user.
-
-         User record in :php:`$GLOBALS['TSFE']->fe_user->user`, if any login.
-         Better use the :ref:`request <t3coreapi:typo3-request>` attribute
-         :ref:`frontend.user <t3coreapi:typo3-request-attribute-frontend-user>` instead.
-
+         .. versionchanged:: 13.0
+            The variable has been removed with TYPO3 v13.
+
+   Migration
```

Comment:
I would make a linkable subheadline here as we might have to link here from other places?

Reply:
Done